content/docs/user-guide/gems/reference/multiplayer/multiplayer/tutorial_multiplayer.md

```diff
@@ -0,0 +1,887 @@
+---
+title: Tutorial - Your First Network Component
+linktitle: Tutorial Multiplayer
+description: Use Multiplayer Gem to create multiplayer components.
+toc: true
+---
+
+
+
+## Introduction
+
+
+This tutorial will link all the relevant documentation to provide you with a detailed guide on getting started with creating multiplayer components.
+
+Our starting base will be O3DE MultiplayerSample project. You can find the instructions on how to build and run it here: https://github.com/o3de/o3de-multiplayersample/blob/development/README.md
+
+
+
+
+
+## Network Components
+
+### Add a New Component
+
+We will start by creating a network component - MyFirstNetworkComponent. It will only do one thing, which is to replicate a monotonic counter.
```

Comment:
Is there real-world benefit for sending round trip uptime between auth and clients? Seems like it could be used for prediction? If there is real-world use could you provide an example?

Reply:
No, there is not any real use. I chose it for its extreme simplicity. Everyone in development space knows about incremental counters. So it should be easy to understand. We can tackle more useful game logic in a different follow-up tutorial that builds on this tutorial.